test/test_scanner.py

```diff
@@ -206,7 +207,17 @@ def condensed_filepath(self, url, package_name):
     @pytest.mark.parametrize(
         "url, package_name, product, version",
         (
-            (d["url"], d["package_name"], d["product"], d["version"])
+            pytest.param(
+                d["url"],
+                d["package_name"],
+                d["product"],
+                d["version"],
+                marks=pytest.mark.skipif(
+                    d["product"] in DISABLED_LONG_TESTS
+                    and os.environ["ACTIONS"] == "1",
```

Comment:
```suggestion
                    or os.environ["ACTIONS"] == "1",
```
I think we want this to be an or, since then people could use this same thing to disable tests locally while testing without having to pretend they're running in GitHub Actions.  Is it possible to have different messages based on which condition triggers the skip?